# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/nikolaus/unison/tail.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -ruN unison-2.9.20/Makefile.OCaml unison-2.9.20-new/Makefile.OCaml
--- unison-2.9.20/Makefile.OCaml	2002-08-19 23:15:04.000000000 +0200
+++ unison-2.9.20-new/Makefile.OCaml	2004-05-24 12:43:49.546990904 +0200
@@ -58,7 +58,7 @@
 # to Unix '/' separators, and extraneous control-M's are deleted.
 # Unfortunately there is a literal control-M buried in this, I'd rather
 # get rid of it...
-OCAMLLIBDIR=$(shell ocamlc -v | tail -1 | sed -e 's/.* //g' | sed -e 's/\\/\//g' | sed -e 's///g')
+OCAMLLIBDIR=$(shell ocamlc -v | tail -n 1 | sed -e 's/.* //g' | sed -e 's/\\/\//g' | sed -e 's///g')
 
 # User interface style: 
 #   Legal values are
